Code-Hammers/code-hammers
# Production Docker Prep and Dev Image Updates

### Description

This PR updates relevant files to prepare for Docker production builds.

### Jira Task

N/A

### Testing Instructions

1. Start up the production container using Dockerfile and ensure the dev container still functions as expected using npm run docker-dev as well.

### Checklist


### Additional Notes, Images, etc.

N/A
